ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-ref.yaml

```diff
@@ -11,9 +11,9 @@ ref:
       memory: 100Mi
   env:
     - name: COMPUTE_NODE_TYPE
-      default: "null"
+      default: "m4.xlarge"
```

Comment:
Isn't this moving us away from deferring to the installer default?  Why is using the installer's default not appropriate?

Reply:
See 0538114746d3b6e1c9ff33bf00051ed2d0363e0f from April 22 of last year.  I consider this definitive.  step registyr regressed cluster-installer-e2e which was case law.

Reply:
On GCP, the installer was [already defaulting to `n1-standard-4`][1] compute, so this PR is a no-op there.  On AWS, it was [defaulting to `*.large`][2], so this raises us above the default.  In order of decreasing preference, it would be nice to test against:

1. Our [documented minimum requirements][3], but that is failing on updates with persistent volumes for Prometheus based on [rhbz#1913532][4].
2. The installer default, but that is failing on AWS updates because those defaults are still too small for the Prometheus issue.
3. The lowest possible value above the installer default that passes.  But this means that users who install without overriding the installer defaults, and then do the sorts of things we're doing in CI, are in a space that we are not watching in CI.

In this case, looking at the compute instance types chosen by Telemetry-reporting customers convinces us that we can raise the default CI type to m4.xlarge without leaving too many customer clusters with smaller compute uncovered by our development CI.

Explicitly setting defaults here also means we are dropping CI coverage for the installer's compute instance-type defaulting logic.  In follow-up work we should adjust a job or add new jobs to restore that coverage.

[1]: https://github.com/openshift/installer/blob/d237b9dee575c6e9ceeba80fd3e137231d907cab/pkg/asset/machines/worker.go#L104
[2]: https://github.com/openshift/installer/blob/d237b9dee575c6e9ceeba80fd3e137231d907cab/pkg/asset/machines/worker.go#L159
[3]: https://docs.openshift.com/container-platform/4.6/installing/installing_bare_metal/installing-bare-metal.html#minimum-resource-requirements_installing-bare-metal
[4]: https://bugzilla.redhat.com/show_bug.cgi?id=1913532